src/main/java/seedu/address/logic/parser/TaTrackerParser.java

```diff
@@ -6,8 +6,11 @@
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 
+import seedu.address.logic.commands.AddCommand;
+import seedu.address.logic.commands.ClearCommand;
 import seedu.address.logic.commands.Command;
 import seedu.address.logic.commands.CommandWords;
+import seedu.address.logic.commands.DeleteCommand;
 import seedu.address.logic.commands.ExitCommand;
 import seedu.address.logic.commands.FindCommand;
 import seedu.address.logic.commands.HelpCommand;
```

Comment:
instead of this long list of imports, why not use `import seedu.address.logic.commands.CommandWords`